### PR TITLE
 modify apidoc briefly for v3

### PR DESCRIPTION
**Pull request by hewonjeong**
- modify apidoc for v3

### CODE REVIEW

File: src/server/items/routes.js [hunk 1]
```diff
@@ -174,11 +186,15 @@ router.get('/:id', ItemControllers.getById);
...
  *      {
- *          "title" : "texttitle",
+ *          "title"       : "texttitle",
  *          "lat"         : 30.565398,
  *          "lng"         : 126.9907941,
  *          "address"     : "testAddress",
  *          "category"    : 'warning', 'event', 'facility'
+ *          "startTime"   : '2016-10-04T04:00:00.578Z',
+ *          "endTime"     : '2016-10-10T04:00:00.578Z'
+ *          "startTime"   : '2016-10-04T04:00:00.578Z',
```
Comment (festinalente91) [inline src/server/items/routes.js:196]: seems like duplicated

File: src/server/items/routes.js [hunk 2]
```diff
@@ -58,14 +62,20 @@ router.get('/', ItemControllers.getAll);
  * @apiParam {Number} lng lng(e.g. 126.9907941)
  * @apiParam {String} address address
  * @apiParam {String} category category
- *
+ * @apiParam {String} startTime startTime
+ * @apiParam {String} endTime endTime
+ * @apiParam {String} image base64 encoding image
```
Comment (ovekyc) [inline src/server/items/routes.js:67]: image (base64 encoded string) 
is better

**Comment by ovekyc** [comment]
LGTM

**Comment by yuda1124** [comment]
LGTM

File: src/server/items/routes.js (hunk 1 above)
Comment (hewonjeong) [reply src/server/items/routes.js:196]: thx, solved!

**Comment by festinalente91** [comment]
LGTM

**Pull request merged**